index.html

```diff
@@ -64,5 +66,182 @@ <h1>I’m MERCY <br />Glad to see <br />you!</h1>
+      <div class="cards">
+        <img src="/assets/images/project4.png" alt="project1" />
+      </div>
+      <div class="cards-details">
+        <div>
+        <h2 class="title">Tonic</h2>
+        <div class="highlights">
+          <p class="highlight-text">Canopy</p>
+            <img class="counter" src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">Back End Dev</p>
+            <img src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">2015</p>
+        </div>
+        </div>
+        <p class="cards-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="grid-skills">
+          <div class="item1">html</div>
+          <div class="item2">css</div>
+          <div class="item3">javascript</div>
+        </div>
+        <button id="popup">see project</button>
+      </div>
+    </section>
+    <!--About me-->
+    <section class="heading1" id="about">
+      <div class="header">
+        <h1>I’m MERCY</h1>
+      </div>
+      <div class="heading-paragraph">
+        <p>
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="heading-social">
+        <div class="icons-header">
+          <p>LET’S CONNECT</p>
+        </div>
+        <ul class="icons">
+          <li id="twitter">
+            <a href="https://twitter.com/MercyMugambi15" target="_blank"
+              ><img src="./assets/icons/twitter.png"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/mercymugambi/" target="_blank"
+              ><img src="./assets/icons/Linkedin.png"
+            /></a>
+          </li>
+          <li>
+            <a href="https://github.com/mercymugambi" target="_blank"
+              ><img src="/assets/icons/github.png"
+            /></a>
+          </li>
+          <li>
+            <a href="#medium" target="_blank"
+              ><img src="./assets/icons/medium.png"
+            /></a>
+          </li>
+          <li>
+            <a href="#angellist" target="_blank"
+              ><img src="./assets/icons/angellist.png"
+            /></a>
+          </li>
+        </ul>
+        <button id="popup" class="resume">Get my resume</button>
+      </div>
+      <div>
+        <p class="lang">Languages</p>
+        <img src="/assets/images/arrow-down.png" alt="img"class="arrow-down"> 
+      </div>
+      <div class="grid-languages">
+        <div class="item1" >
+          <img src="/assets/images/js.png" alt="js">html
+        </div>
+        <div class="item2"><img src="/assets/images/html.png" alt="html">css</div>
+        <div class="item3"><img src="/assets/images/css.png" alt="css">javascript</div>
```

Comment:
This is great but it could be improved with some spacing and by centering the image with the text. See below 👇🏽 

## Figma
![Screenshot 2023-02-17 at 21 40 57](https://user-images.githubusercontent.com/45739963/219827554-bbc1ae6e-8c76-41f4-ba4a-48a0c16796dd.png)


## Your design
![Screenshot 2023-02-17 at 21 41 19](https://user-images.githubusercontent.com/45739963/219827566-684e8273-a09c-48a8-a34b-53148e0ef3eb.png)